server/routes/serviceProviderReferrals/serviceProviderReferralsController.test.ts

```diff
@@ -269,10 +269,7 @@ describe('GET /service-provider/dashboard/all-open-cases', () => {
       .expect(200)
       .expect(res => {
         expect(res.text).toContain('All open cases')
-        expect(res.text).not.toContain('Alex River')
-        expect(res.text).not.toContain('Accommodation Services - West Midlands')
-        expect(res.text).not.toContain('George River')
-        expect(res.text).toContain(`There are no results for`)
+        expect(res.text).toContain('Date received') // Proves tables existence
```

Comment:
The test is named "all open cases and no cases when there are no open cases to be viewed" -- I wonder if it should be renamed to cover what the expectation is?